App/Sagas/DeviceLogs.ts

```diff
@@ -0,0 +1,13 @@
+import {put, select} from 'redux-saga/effects'
+import DeviceLogsActions from '../Redux/DeviceLogsRedux'
+import {PreferencesSelectors} from '../Redux/PreferencesRedux'
+
+export function * logNewEvent (time: number, event: string, message: string, error?: boolean) {
+  if (yield select(PreferencesSelectors.verboseUi)) {
```

Comment:
Same comment... Seems like we should always log so the data is there when you turn verbose UI on.

Reply:
intentional for now. i don't want to maintain any logs or deal with this flow on user's devices that aren't running verbose. let's go with it like this and if we see cases where it's really required without verbose ui we can sort it

Reply:
Cool